.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "SpokeStack/filter_audio"]
+	path = SpokeStack/filter_audio
+	url = git@github.com:noelweichbrodt/filter_audio.git
```

Comment:
it would be better if this was forked into pylon instead of your personal account